docs/bidders/emx_digital.md

```diff
@@ -0,0 +1,10 @@
+# EMX Digital Bidder
+
+To use this bidder you will need a tagid from our exchange, [https://emxdigital.com/](EMX Digital).
+
```

Comment:
This is very minor but, don't you want to send traffic to the exact URL where they can get a new `tagid` instead of your frontpage? This is just a suggestion and it's entirely up to you.

Reply:
That makes sense, we have reworded this Doc to be more clear. Thanks!